.github/workflows/run-example-deployment.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
-#        with:
+#        with: TODO what to do with this? Remove it?
 #          path: "github-action-deploy-on-vercel" # Act doesn't work if not specifying the path. See https://github.com/nektos/act/issues/228
```

Comment:
Yes we can remove it because ACT is not implemented yet (and I guess it'll never)
```suggestion
```